xml/System/GCMemoryInfo.xml

```diff
@@ -19,7 +19,7 @@
     </Attribute>
   </Attributes>
   <Docs>
-    <summary>To be added.</summary>
+    <summary>A collection of APIs that can be used to retrieve garbage collection information.</summary>
```

Comment:
I added this summary. If it does not look clear enough, feel free to extend it or change it.